conjur/api/api.py

```diff
@@ -221,6 +225,28 @@ def get_variables(self, *variable_ids) -> dict:
 
         return remapped_keys_dict
 
+    def create_token(self, create_token_data: CreateTokenData) -> requests.Response:
+        """
+        This method is used to create token/s for hosts with restrictions.
+        """
+        if create_token_data is None:
+            raise MissingRequiredParameterException('create_token_data')
+        params = {}
+        params.update(self._default_params)
+
+        token_params = {}
+        for attr, value in create_token_data.__dict__.items():
```

Comment:
if we use __repr__ function only here maybe we could set it to return only fields that have values?